game/configuration.cc

```diff
@@ -352,6 +392,21 @@ void readConfigXML(fs::path const& file, int mode) {
 	}
 }
 
+int PaHostApiNameToHostApiTypeId (const std::string& name) {
+	if (name == "Auto") return 1337;
+	if (name == "Windows DirectSound") return 1;
+	if (name == "MME") return 2;			
+	if (name == "ASIO") return 3;
+	if (name == "Core Audio" || name == "CoreAudio") return 5;
+	if (name == "OSS") return 7; // Not an error, stupid PortAudio.
+	if (name == "ALSA") return 8;
```

Comment:
I'm not sure I'm following; can you be more precise? "Auto" is meant to fullfill two roles:

1) Fallback in case of invalid settings (for example, config file exported from one machine or platform to another), as well as a sensible default for casual end-users. i.e. the option that should work most of the time; the way it does this is simply by selecting (at runtime) an audio backend that should definitely be available depending on which platform we are running. This will be CoreAudio for macOS, ALSA for Linux (this might be subject to change since I'm not really all that familiar with different linux distros) and WASAPI for Windows.

The only thing this does is instruct performous how it should access audio devices; it should not really change which audio devices are selected by default. IIRC, in absence of a sensible choice, performous looks for the first audio device with 2 or more out channels and uses that for playing sound, correct?

So... "audio/backend" should be set to "Auto" most of the time, unless the user is sure of what he's doing (either way, the UI should never allow him to select an invalid configuration, manually editing the xml notwithstanding obviously.)

So, if you mean something related to what "audio/backend" displays in that case, odds are it's correct. If you are referring to something in the Audio Devices screen, it might be a bug that I missed.